doc/src/site/sphinx/ml/sw_target_encoder.rst

```diff
@@ -32,55 +32,6 @@ An example of converting a categorical feature to continues with Target Encoder
 Target Encoding can help to improve accuracy of machine learning algorithms when columns with high
 cardinality are used as features during a training phase.
 
-Parameters
```

Comment:
What  do you think of adding link to generated parameters doc?

Reply:
I wanted to do it, but forgot about that. Good catch, thanks!